Documentation/installation/cli-download.rst

```diff
@@ -1,3 +1,13 @@
+.. warning::
+  Make sure you install `cilium-cli v0.15.0 <https://github.com/cilium/cilium-cli/releases/tag/v0.15.0>`_
+  or later. The rest of instructions do not work with older versions of
+  cilium-cli. To confirm the cilium-cli version that's installed in your system,
+  run:
```

Comment:
Nit. It might make sense to add a link to the specific section of the upgrade notes.

Reply:
i'll update it before marking it ready to merge 📝

Reply:
✨ https://deploy-preview-26606--docs-cilium-io.netlify.app/gettingstarted/k8s-install-default.html#install-the-cilium-cli ✨